frontend/src/components/SamplesList.tsx

```diff
@@ -96,10 +96,7 @@ export default function SamplesList({
     });
 
   const samples = data?.dashboardSamples;
-  const sampleCount =
-    data?.dashboardSamples[0] !== undefined
-      ? data?.dashboardSamples[0]._total
-      : 0;
+  const sampleCount = data?.dashboardSamples[0]._total || 0;
```

Comment:
Could we add the missing `?.` operator here? We can merge otherwise. Thanks!
```ts
const sampleCount = data?.dashboardSamples[0]?._total || 0;
```

Reply:
So nothing breaks when there is 0 records returned.

Reply:
added